share/github-backup-utils/ghe-s3-backup

```diff
@@ -22,5 +22,4 @@ ghe-backup
 s3cmd mb s3://$GHE_S3_BUCKET
 
 # Upload to S3.
-cd "$GHE_DATA_DIR"/current
-s3cmd --preserve put * s3://$GHE_S3_BUCKET
+s3cmd sync --delete-removed --follow-symlinks "$GHE_DATA_DIR"/current/ s3://$GHE_S3_BUCKET
```

Comment:
Since this command is kind of foreign and the command line flags are certainly unique, could you add a comment section above this to describe what some of these flags are doing?